setup.py

```diff
@@ -1,27 +1,26 @@
 import os
 from setuptools import setup, find_packages
-version = '0.3.0'
+version = '1.0.0'
 README = os.path.join(os.path.dirname(__file__), 'README')
 long_description = open(README).read()
 setup(name='locking',
       version=version,
+      packages=find_packages(),
+      install_requires=['django-staticfiles','simplejson'],
```

Comment:
Do we still need `django-staticfiles` here? It seemed to install one that is Python 2 version for me. Also, I don't think we need `simplejson` here.

Reply:
I'll check and update.

Reply:
I tried running this app, migrations and tests without these and it work just fine. I'll get rid of this from setup.py